server/src/yml-objects/AbstractYmlFunction.ts

```diff
@@ -0,0 +1,10 @@
+import { AbstractYmlObject, YmlType } from './AbstractYmlObject';
+import { YmlArgs } from './YmlArgs';
+import { YmlSymbolList } from './YmlSymbolList';
+
+export abstract class AbstractYmlFunction extends AbstractYmlObject {
+    public returnDomains: YmlType;
+    public returnDomainsLevel2: YmlType;
```

Comment:
Consider adding a comment explaining what “domains of level 2” are, since this question is bound to come up every now and then.